serde-reflection/src/lib.rs

```diff
@@ -8,10 +8,10 @@
 //!
 //! Format descriptions are useful in several ways:
 //! * Stored under version control, formats can be tested to prevent unintended modifications
-//! of binary serialization formats (e.g. by changing variant order).
+//!   of binary serialization formats (e.g. by changing variant order).
```

Comment:
Can you avoid modifying unrelated files please?

Reply:
Ok, reverted those corrections induced by clippy.